ngraph_bridge/ngraph_encapsulate_impl.h

```diff
@@ -85,22 +85,42 @@ class NGraphEncapsulateImpl {
       const ng::element::Type& ng_element_type, const ng::Shape& ng_shape,
       std::shared_ptr<ng::runtime::Tensor> tensor_from_pipeline);
 
-  // Accessors(getters and setters) for the private data members of
-  // NgraphEncapsulateImpl class
-  // needed by
-  // NgraphEncapsulateOp class
-  const int& GetNumberOfCopies() { return number_of_copies; }
+  // Get pipeline index and input and output tensor groups from executable (if
+  // they can create tensors)
+  Status GetPipelineIdxAndTensors(
+      const std::shared_ptr<ngraph::runtime::Executable>&,
+      std::tuple<int, PipelinedTensorVector, PipelinedTensorVector>&);
+
+  // Once done using, return the index to indicate that those executable created
+  // tensors are free for reuse
+  Status ReturnPipelinedTensors(std::shared_ptr<ngraph::runtime::Executable>,
+                                size_t);
+
+  // Clear all maps with ng_exec as keys
+  void ClearExecMaps();
+
+  // Parse additional attributes attached to the encapsulate node and filter out
+  // the ones needed for backend->SetConfig
+  Status ParseNodeAttributes(
+      const google::protobuf::Map<string, AttrValue>& additional_attributes,
+      std::unordered_map<std::string, std::string>* additional_attribute_map);
+
+// Accessors(getters and setters) for the private data members of
+// NgraphEncapsulateImpl class
+// needed by NgraphEncapsulateOp class
 
+#if defined(NGRAPH_TF_ENABLE_VARIABLES_AND_OPTIMIZERS)
```

Comment:
These 5 functions are only applicable for varopts build, so moved them under an ifdef